segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/stat/AutoScalerConfig.java

```diff
@@ -27,6 +27,12 @@
     public static final Property<Integer> CACHE_CLEANUP_IN_SECONDS = Property.named("cacheCleanUpInSeconds", 5 * 60);
     public static final Property<Integer> CACHE_EXPIRY_IN_SECONDS = Property.named("cacheExpiryInSeconds", 20 * 60);
     public static final Property<String> CONTROLLER_URI = Property.named("controllerUri", "tcp://localhost:9090");
+    public static final Property<Boolean> TLS_ENABLED = Property.named("tlsEnabled", false);
```

Comment:
Is it correct that this property is no longer used?

Reply:
Same as above but for segmentstore.

Reply:
I thought that the tls setting in AutoScalerConfig was for segment store as a client of controller, and  `ServiceConfig` was where the tlssetting for segment store as a server was defined?

In other words, if I turn off ssl in the controller's `application.conf` file, then any client talking to it can indicate whether or not it wants to talk ssl using the scheme (as per the latest changes).   So I thought that meant the tls enabled in one of `AutoScalingConfig` or `ServiceConfig` is obsolete?